app/components/sections/hero_component.html.erb

```diff
@@ -1,17 +1,22 @@
-<%= tag.div(class: classes) do %>
-  <header>
-      <div class="hero__titles">
-        <div class="hero__titles__title">
-          <h1><%= tag.span(title) %></h1>
+<%= tag.header(class: classes) do %>
+    <%= responsive_image %>
+    <div class="hero__title">
+      <h1><%= tag.span(title) %></h1>
+    </div>
+      <% if subtitle.present? %>
+        <div class="hero__subtitle">
+          <%= tag.div(subtitle) %>
+        </div>
+      <% end %>
+      <% if false %>
```

Comment:
Did you mean to leave this in?

Reply:
Ah no, was experimenting with sample content, will remove.